### PR TITLE
Polkadot Protocol > Protocol Components > Accounts > Account Balance Types

### PR DESCRIPTION
**Pull request by kapetan3sid**
Migrated https://wiki.polkadot.network/docs/learn-account-balances page

### CODE REVIEW

File: .DS_Store
Comment (CrackTheCode016) [inline .DS_Store:1]: Remove please

File: images/.DS_Store
Comment (CrackTheCode016) [inline images/.DS_Store:1]: ditto

File: images/polkadot-protocol/.DS_Store
Comment (CrackTheCode016) [inline images/polkadot-protocol/.DS_Store:1]: ditto

File: images/polkadot-protocol/protocol-components/.DS_Store
Comment (CrackTheCode016) [inline images/polkadot-protocol/protocol-components/.DS_Store:1]: ditto

File: images/polkadot-protocol/protocol-components/accounts/.DS_Store
Comment (CrackTheCode016) [inline images/polkadot-protocol/protocol-components/accounts/.DS_Store:1]: ditto

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (CrackTheCode016) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:13]:
```suggestion
This update will enable more efficient use of account balances, allowing the free balance to be utilized for on-chain activities such as setting proxies and managing identities.
```

**Review by CrackTheCode016 — APPROVED** [review]
lgtm, just some minor stuff to remove before merging

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:21]:
```suggestion
- **Free** - the balance available for on-chain activities such as staking and participating in governance, but not necessarily spendable or transferable
- **Frozen** - the portion of the free balance that is locked for specific purposes like [staking](path to staking page), [governance](path to governance page), or [vesting](path to Vested Transfers subsection in Types of Txs) (also referred to as locked balance)
- **On Hold** - the balance reserved for [identities](path to Account Identities section), [proxies](path to Proxy Accounts page), [OpenGov preimages and deposits](path to OpenGov deposits page). It is no longer free and is sometimes called reserved balance
- **Spendable** - the free balance that is fully available for spending or transferring
- **Untouchable** - the part of the free balance that cannot be transferred or spent, but can still be used for other on-chain activities
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:31]: These are the terms that most people know about. In the list above, I think it is better to replace On Hold with Reserve and Frozen with Locked. What do you think?

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:40]:
```suggestion
Locks are included in the account's `frozen` balance, which represents funds that may be free but are non-transferable. This balance is tied up in activities such as [staking](path to staking page), [governance](path to governance page) and [vesting](path to Vested Transfers subsection in Types of Txs).
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:45]:
```suggestion
1. If multiple locks involve different amounts of tokens, the largest lock determines the total amount of locked tokens
2. If multiple locks involve the same amount of tokens, the lock with the longest duration dictates when those tokens can be unlocked
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:65]:
```suggestion
1. First, you must remove the governance lock on 24 DOT after 7 days

2. Next, you can unlock the 80 DOT from staking after 28 days

3. Finally, after 224 days, you will be able to remove the second governance lock
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:84]: Using `here` for a link is not a good practice. Replace with a more self-descriptive name. It could be something like Check this [post]() on Stack Exchange or smth like that

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:81]: We don't use exclamation marks
```suggestion
You will get a 6x conviction for 24 DOT
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:88]:
```suggestion
## Balance Types on Polkadot.js

The following example displays the different balance types on the Polkadot.js UI (wallet) for a Kusama account. 
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:111]:
```suggestion
- The **total balance** represents the entire number of tokens in the account, but it doesn’t necessarily reflect the tokens you can transfer. In this example, the total balance is **0.6274 KSM**

- The **transferrable balance** shows the number of tokens available for transfer. It is calculated by subtracting the locked and reserved tokens from the total balance. In the example, the transferrable balance is 0.0106 KSM

- **Locked funds** refer to tokens used for staking, governance, or vested transfers (details below)

- The **vested balance** refers to tokens sent to the account that are released according to a specific time schedule. Although the account owns these tokens, they are locked and will only become transferable after a certain number of blocks. In this example, the vested balance is **0.25 KSM**

- The **bonded balance** represents the number of tokens locked for on-chain activities such as staking. In this example, the bonded balance is **0.4 KSM**

- The **democracy balance** represents the number of tokens locked for on-chain participation in governance activities, such as voting on referenda and council elections. In this example, the democracy balance is **0.4 KSM**

- The **redeemable balance** shows the number of tokens that have completed the unbonding period and are ready to be unlocked, becoming transferable again. In this example, the redeemable balance is **0.1 KSM**

- The **locked balance** represents the number of frozen tokens due to participation in staking, democracy, or vested transfers. **Locks do not stack**, meaning the total locked balance is determined by the largest lock, not the sum of individual locks

- The **reserved balance** represents tokens frozen for on-chain activities other than staking, governance, or vested transfers. This can include actions like setting an identity or a proxy. Reserved funds are locked due to on-chain requirements but can typically be freed by taking specific on-chain actions. For instance, the "Identity" pallet reserves funds while an on-chain identity is registered, but by clearing the identity, those funds can be unreserved and made available again. The same applies to proxies, which reserve funds due to their memory usage on the network. In this example, a governance proxy was created, and the reserved balance is 0.0668 KSM
```

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (0xLucca) [inline polkadot-protocol/protocol-components/accounts/account-balance-types.md:105]: Democracy balance should be 0.55 KSM.

File: polkadot-protocol/protocol-components/accounts/account-balance-types.md
Comment (kapetan3sid) [reply polkadot-protocol/protocol-components/accounts/account-balance-types.md:31]: That makes sense, I will amend it

**Review by 0xLucca — APPROVED** [review]
LGTM

**Review by CrackTheCode016 — APPROVED** [review]
lgtm, nice work

**Comment by eshaben** [comment]
@dawnkelly09 - please check this over to make sure I didn't miss anything 🙂

**Review by CrackTheCode016 — APPROVED** [review]

**Review by CrackTheCode016 — APPROVED** [review]

**Review by dawnkelly09 — APPROVED** [review]
LGTM!

**Review by nhussein11 — APPROVED** [review]
LGTM!

**Pull request merged**
